assignment/src/com/assignment3/Files/Files.java

```diff
@@ -9,28 +9,28 @@
  */
 public class Files {
     //Creating objects for I/O streaming classes
-    static FileInputStream fileInputStream =null;
-    static FileOutputStream fileOutputStream = null;
-    static FileInputStream fileInputStream1 = null;
+    public static FileInputStream FILE_INPUT_STREAM =null;
+    public static FileOutputStream FILE_OUTPUT_STREAM = null;
+    public static FileInputStream FILE_INPUT_STREAM1 = null;
```

Comment:
Reference names should not start with capital letters. No need of creating constants for object reference.
Create constant for file names - C:\\Workspace\\repo-java\\assignment\\src\\com\\assignment3\\Files\\source_file.txt